### PR TITLE
Fixed the connection issue between KinD and Microshift

### PR DESCRIPTION
**Pull request by yanmxa**
fixes: https://github.com/stolostron/backlog/issues/23991
Signed-off-by: myan <myan@redhat.com>

### CODE REVIEW

File: test/setup/microshift/microshift_setup.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    sleep 5;
+  done;
+  docker cp $clusterName:/var/lib/microshift/resources/kubeadmin/kubeconfig $currentDir/../config/kubeconfig-microshift
+  echo "containerIp=$(docker inspect -f '{{.NetworkSettings.IPAddress}}' $clusterName)"
+  containerIp=$(docker inspect -f '{{.NetworkSettings.IPAddress}}' $clusterName)
+  sed "s;127.0.0.1;${containerIp};g" $currentDir/../config/kubeconfig-microshift > $KUBECONFIG
+fi 
+
+kubectl wait deployment -n openshift-ingress router-default --for condition=Available=True --timeout=600s
+kubectl wait deployment -n openshift-service-ca service-ca --for condition=Available=True --timeout=600s
```
Comment (morvencao) [inline test/setup/microshift/microshift_setup.sh:21]: when router and service-ca are ready, the microshift cluster is ready?

Comment (yanmxa) [reply test/setup/microshift/microshift_setup.sh:21]: The Microshift will take a long time to be ready. I am not sure when the cluster is available. So currently I use the service ca and router status to determine that the cluster is ready to install other components.


That would be great if you have other options to confirm the cluster is available!

**Review by morvencao — COMMENTED** [review]
/lgtm

**Pull request merged**